lntest/itest/lnd_test.go

```diff
@@ -6381,8 +6381,8 @@ func subscribeChannelNotifications(ctxb context.Context, t *harnessTest,
 
 // verifyCloseUpdate is used to verify that a closed channel update is of the
 // expected type.
-func verifyCloseUpdate(chanUpdate *lnrpc.ChannelEventUpdate,
-	force bool, forceType lnrpc.ChannelCloseSummary_ClosureType) error {
+func verifyCloseUpdate(chanUpdate *lnrpc.ChannelEventUpdate, force bool,
+	coopType, forceType lnrpc.ChannelCloseSummary_ClosureType) error {
```

Comment:
could these three vars be condensed down to just one single `closeType lnrpc.ChannelCloseSummary_ClosureType`?

Reply:
+1